### ISSUE TITLE
Request: Dates returned as datetime objects

### ISSUE THREAD

**Issue by tobiasbp** [kanboard/python-api-client#13]
Dates are returned as strings. The should be returned as datetime objects.

Here is the data from a project:

{'priority_end': '3', 'is_everybody_allowed': '0', 'is_public': '0', 'owner_id': '0', 'predefined_email_subjects': None, 'show_default_swimlane': '1', 'is_private': '0', 'priority_start': '0', 'default_swimlane': 'Default swimlane', 'id': '29', 'token': '', 'end_date': '2019-08-18', 'url': {'board': 'https://kanboard.kontrapunkt.com/board/29', 'list': 'https://kanboard.example.com/list/29'}, 'last_modified': '1558972198', 'email': '', 'identifier': '****', 'name': '**', 'start_date': '2019-02-19', 'priority_default': '0'}

start_date & end_date should be datetime objects.

**Comment by fguillot** [comment]
Yes, it's a little bit messy, the `last_modified` field could be also a datetime object. The reason is the same as #12.

Maybe the Python client could be modified to convert date fields.

**Comment by tobiasbp** [comment]
> Maybe the Python client could be modified to convert date fields.

Yes, that is what I'm suggesting. Is it something you would add to python-client-api if I add it?
